templates/post.html

```diff
@@ -91,6 +91,25 @@ <h3 class="p-media-object__title">
     </div>
   </div>
 </div>
+{% for tag in tags %}
+  {% if tag.slug == 'robotics' %}
+    <div class="p-strip--cta">
```

Comment:
Please use: `<div class="p-strip--image is-dark" style="background-image: linear-gradient(44deg, #171717 0%, #181818 9%, #262626 34%, #2D2D2D 67%, #383838 88%, #2E2E2E 100%, #393939 100%);">` as its just applying a background-image. This would mean there would be no need for a custom pattern.